### DISCUSSION TITLE
Installation instructions in README.md don't create persistent volume storage.

### DISCUSSION THREAD

**Discussion by Zcehtro** [dani-garcia/vaultwarden#4756] [Q&A]
### Subject of the issue

The command `docker run -d --name vaultwarden -v /vw-data/:/data/ --restart unless-stopped -p 80:80 vaultwarden/server:latest` does not create a persistent volume.

Persistent storage is not implemented on creation of the container.

### Deployment environment

* vaultwarden version: 1.31.0

* Install method: docker image

* Other relevant details: ran via powershell core on Windows 10

### Steps to reproduce

Run command found on `README.md` :

```sh
docker run -d --name vaultwarden -v /vw-data/:/data/ --restart unless-stopped -p 80:80 vaultwarden/server:latest
```

### Expected behaviour
Creates the container and creates the docker volume with the appropriate binding.

### Actual behaviour

Creates the container and does not create the volume.

### Troubleshooting data

Fixed command that worked for me is on powershell core on Windows 10:

```sh
docker run -d --name vaultwarden -v vw-data:/data/ --restart unless-stopped -p 80:80 vaultwarden/server:latest
```

Basically remove the forward slashes from `vw-data`.

**Comment by BlackDex** [comment]
That is because you do not have a `/vw-data` directory on your host. The command works just fine.

**Reply by Zcehtro** [reply]
I don't understand the reasoning behind this. Could you please elaborate? 

Under which circumstance should I already have a `/vw-data` directory on my host?

**Reply by BlackDex** [reply]
You probably never have. Same as a `vw-data/` folder. But since you probably have rights to create a folder there it does that automatically. And you probably do not have normal user rights on your `/` root directory.

**Reply by Zcehtro** [reply]
I think I understand.

However, the point I was making with my ticket is that with `vw-data` without forwards slashes will create the folder and volume automatically if it does not exist, while `/vw-data/` will not.

**Reply by BlackDex** [reply]
It will if you have to correct rights.

**Reply by tessus** [reply]
@Zcehtro just to be clear: without any slashes a volume is created, with slashes a directory. See the differences between bind mounts and docker volumes.
